golangci.yaml

```diff
@@ -9,3 +9,11 @@ linters:
     - staticcheck
     - typecheck
     - unused
+issues:
```

Comment:
Adding specific linter exclude directives. I can ignore just SA1019, but I think this is safer. I also think doing this here is cleaner than putting `lint:ingore` all over the code.